src/Test/QuickCheck/Data/ApproxNumber.purs

```diff
@@ -1,13 +1,21 @@
-module Test.QuickCheck.Data.ApproxNumber where
+module Test.QuickCheck.Data.ApproxNumber
```

Comment:
It's used when you want to generate numeric values that will actually be comparable :smile: (usually inside another structure)